kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkRollOverStrategyImpl.java

```diff
@@ -1,43 +0,0 @@
-package com.slack.kaldb.chunkManager;
```

Comment:
Let's keep the existing strategy as is. Let's make a new roll over strategy and use it in the code. The roll over strategy code is already configurable.